x11-libs/vte/files/vte-0.76.3-stdint.patch

```diff
@@ -0,0 +1,15 @@
+
+Fix compilation with gcc-14.
+
+Signed-off-by: Holger Hoffstätte <holger@applied-asynchrony.com>
+
```

Comment:
Half a git metadata? :D  

Can we get an upstream link to https://gitlab.gnome.org/GNOME/vte/-/issues/2807 / https://gitlab.gnome.org/GNOME/vte/-/commit/0d393b6cd6a24f53eaefa16764b9453a1483acf5 instead or in addition?

Reply:
Updated.